src/Command/CodeStudio/CodeStudioDisableDefaultJobCommand.php

```diff
@@ -0,0 +1,281 @@
+      $project_choice_labels = array_values($list);
+      $question = new ChoiceQuestion('Please select a Cloud Platform subscription',
+        $project_choice_labels,
+        $project_choice_labels[0],
+      );
+      $choice_id = $this->io->askQuestion($question);
+      $project_id = array_search($choice_id, $list, TRUE);
+    }
+
+    // Variables list for Code Studio default jobs
+    // https://docs.acquia.com/code-studio/configuring/modifying-default-job/#enabling-or-disabling-a-default-code-studio-job
+    $variables_list = [
+      'ACQUIA_JOBS_BUILD_DRUPAL',
+      'ACQUIA_JOBS_TEST_DRUPAL',
+      'ACQUIA_TASKS_INSTALL_DRUPAL',
+      'ACQUIA_JOBS_DEPLOY_BRANCH',
+      'ACQUIA_JOBS_DEPLOY_TAG',
+      'ACQUIA_JOBS_CREATE_CDE',
+      'ACQUIA_JOBS_DEPRECATED_UPDATE',
+      'ACQUIA_JOBS_COMPOSER_UPDATE',
+      'SAST_DISABLED',
+    ];
+
+    $variable = NULL;
+    if ($input->getOption('gitlab-project-job')) {
+      $variable = $this->input->getOption('gitlab-project-job');
+      // If job provided by user is not valid.
+      if (!in_array($variable, $variables_list)) {
+        $this->io->error('Variable provided is not valid.');
+        return 1;
+      }
+    }
+
+    if (!$variable) {
+      $question = new ChoiceQuestion('Please select the Code Studio default job you want to disable',
+        $variables_list,
+        $variables_list[0],
+      );
+      $choice_id = $this->io->askQuestion($question);
+      $variable = array_search($choice_id, $variables_list, TRUE);
+    }
+
+    // Check if the variable already exists or not.
+    $all_project_variables = $this->gitLabClient->projects()->variables($project_id);
+    $is_variable_exist = FALSE;
+    foreach (array_values($all_project_variables) as $project_variable) {
+      if ($project_variable['key'] === $variable) {
+        $is_variable_exist = TRUE;
+        break;
+      }
+    }
+
+    try {
+      // If  variable not exists in project, create it.
+      if (!$is_variable_exist) {
+        $this->gitLabClient->projects()->addVariable($project_id, $variable, 'false');
+      } else {
+        // If variable already exists, updating the variable.
+        $this->gitLabClient->projects()->updateVariable($project_id, $variable, 'true');
+      }
+    }
+    catch (RuntimeException $exception) {
+      $this->io->error([
+        "Unable to update the job",
+      ]);
+      // Log the error for debugging purpose.
+      $this->logger->debug('Error @error while updating/creating the job @variable for the project @project project id @project_id', [
+        '@error' => $exception->getMessage(),
+        '@project' => $list[$project_id],
+        '@project_id' => $project_id,
+        '@variable' => $variable,
+      ]);
+      return 1;
+    }
+
+    $this->io->info("Job `{$variable}` is disabled for the project `{$list[$project_id]}`");
+    return 0;
+  }
+
+  /**
+   * @return \Gitlab\Client
+   */
+  protected function getGitLabClient(): Client {
```

Comment:
For now added an abstract class for Gitlab which extends the command base.
Not sure how this can be utilized (if to use) `CodeStudioWizardCommand` as `CodeStudioWizardCommand` extends the `WizardCommandBase` while this new command not require wizard command